tests/countries/test_central_african_republic.py

```
@@ -0,0 +1,169 @@
+
+    def test_country_aliases(self):
+        self.assertAliases(CentralAfricanRepublic, CF, CAF)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(CentralAfricanRepublic(years=1958))
+
+    def test_new_years_day(self):
+        name = "Jour de l'an"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1959, 2050)))
+
+    def test_barthélemy_boganda_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)